packages/cosmic-swingset/lib/ag-solo/vats/issuers.js

```diff
@@ -21,76 +20,227 @@ export const CENTRAL_ISSUER_NAME = 'MOE';
  * @property {Brand} [brand]
  * @property {Array<any>} [issuerArgs]
  * @property {CollateralConfig} [collateralConfig]
- * @property {string} pursePetname
- * @property {Bigish} mintValue
- * @property {Array<[Bigish, Bigish]>} [fakeTradesGivenCentral]
+ * @property {Array<[string, Bigish]>} [defaultPurses]
+ * @property {Array<[Bigish, Bigish]>} [tradesGivenCentral]
  */
 
+export const makeScaler = toDecimals => (n, fromDecimals = 0) => {
+  if (typeof n === 'bigint') {
+    return n * 10n ** BigInt(toDecimals);
+  }
+  return (
+    BigInt(Math.floor(n * 10 ** fromDecimals)) *
+    10n ** BigInt(toDecimals - fromDecimals)
+  );
+};
+export const scaleMills = makeScaler(4);
+export const scaleMicro = makeScaler(6);
+export const scaleEth = makeScaler(18);
+export const scaleCentral = scaleMicro;
+
 /** @type {Array<[string, IssuerInitializationRecord]>} */
-const fakeIssuerEntries = [
+const fromCosmosIssuerEntries = [
+  [
+    'BLD',
+    {
+      issuerArgs: [undefined, { decimalPlaces: 6 }],
+      defaultPurses: [['Agoric staking token', scaleMicro(73)]],
+      collateralConfig: {
+        keyword: 'BLD',
+        collateralValue: scaleMicro(1000000n),
+        initialMarginPercent: 150n,
+        liquidationMarginPercent: 125n,
+        interestRateBasis: 250n,
+        loanFeeBasis: 50n,
+      },
+      tradesGivenCentral: [
+        [scaleCentral(27.9, 1), scaleMicro(1)],
+        [scaleCentral(25.7, 1), scaleMicro(1)],
+        [scaleCentral(26.8, 1), scaleMicro(1)],
+      ],
+    },
+  ],
+];
+
+harden(fromCosmosIssuerEntries);
+export { fromCosmosIssuerEntries };
+
+/** @type {Array<[string, IssuerInitializationRecord]>} */
+const fromPegasusIssuerEntries = [
+  [
+    'ATOM',
+    {
+      issuerArgs: [undefined, { decimalPlaces: 6 }],
+      defaultPurses: [['Cosmos Staking', scaleMicro(68)]],
+      collateralConfig: {
+        keyword: 'ATOM',
+        collateralValue: scaleMicro(1000000n),
+        initialMarginPercent: 150n,
+        liquidationMarginPercent: 125n,
+        interestRateBasis: 250n,
+        loanFeeBasis: 50n,
```

Comment:
Dont these need an initial price?

Reply:
IS that `collateralValue`?

Reply:
Initial price comes from `tradesGivenCentral[0]`.  I've added a comment.